package_manifest.yaml

```diff
@@ -65,6 +65,12 @@ copr_projects:
             - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
           buildroot_packages: "{{ core_buildroot_packages }}"
+        - name: "rhel-{{ rhel_9 }}-x86_64"
+          external_repos:
+            - "https://yum.puppet.com/puppet7/el/{{ rhel_9 }}/x86_64/"
+            #- "{{ foreman_staging }}/rhel-{{ rhel_9 }}-x86_64"
```

Comment:
there is no staging repo yet for el9

Reply:
What happens if you go ahead and define it here? As soon as you run this to create the EL9 chroot, this staging repository will exist (since that chroot will trigger Copr to create it).

Reply:
Within this PR it would fail until someone with the right permissions calls `obal copr-project …` to create it.

But yes, this is the plan, I just wanted to see if things otherwise build OK.